assets/helpers/contributions.js

```diff
@@ -21,6 +21,8 @@ export type BillingPeriod = 'Monthly' | 'Annual';
 
 export type Amount = { value: string, spoken: string };
 
+export type PaymentType = 'paypal' | 'card';
```

Comment:
I think it might be clearer to model payment types on this site are Paypal, Stripe and GoCardless